Magical Nights/recipes/blacksmithing.json

```
@@ -25,6 +25,32 @@
     "tools": [ [ [ "demon_forge", 15 ] ], [ [ "tongs", -1 ] ] ],
     "components": [ [ [ "scrap_bronze", 9 ] ], [ [ "mercury", 400 ] ] ]
   },
+  {
+    "result": "mithril_ingot",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 5,
```

```suggestion
    "difficulty": 6,
```
Mithril should probably be more difficult than orichalcum to reflect its better stats for items.